portfolio/src/main/java/com/google/sps/servlets/ListCommentsServlet.java

```diff
@@ -61,16 +66,24 @@ public void doGet(HttpServletRequest request, HttpServletResponse response) thro
   }
 
   /**
-  *@return the max number of comments parameter of the request or DEFAULT_MAX_NUM_COMMENTS if invalid. 
+  *@return the request parameter or default value if invalid. 
```

Comment:
We don't use `@return`in summary fragment. You can change it to 
>Return the request parameter or default value if invalid.

Also in DeleteCommentServlet.

https://google.github.io/styleguide/javaguide.html#s7.2-summary-fragment

Reply:
fixed